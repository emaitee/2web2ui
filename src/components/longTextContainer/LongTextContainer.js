import React from 'react';
import styles from './LongTextContainer.module.scss';

const LongTextContainer = ({ text }) => (
  <div className={styles.LongText}>{text}</div>
);

export default LongTextContainer;
