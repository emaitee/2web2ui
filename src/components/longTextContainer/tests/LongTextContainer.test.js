import React from 'react';
import LongTextContainer from '../LongTextContainer';
import { shallow } from 'enzyme';

describe('Long Text Container: ', () => {
  it('should render', () => {
    const wrapper = shallow(<LongTextContainer text={'foo'} />);
    expect(wrapper).toMatchSnapshot();
  });
});
