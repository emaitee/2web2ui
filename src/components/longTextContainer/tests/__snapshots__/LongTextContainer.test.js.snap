// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Long Text Container:  should render 1`] = `
<div>
  foo
</div>
`;
